testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/components/teaser/v1/Teaser.java

```diff
@@ -45,6 +47,14 @@ public boolean isImagePresent(String path) {
         return $(String.format(imageTag, path)).isDisplayed();
     }
 
+    public boolean isImagePresentWithFileName(String fileName) {
+        return $(String.format(imageWithFileName, fileName)).isDisplayed();
+    }
+
+    public boolean isImagePresentWithAtlText(String pagePath, String altText) {
```

Comment:
`isImagePresentWithAltText`

Reply:
@richardhand Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1828/commits/484eef1b066e43e55d44e6ffd343f556de31bccb